action.sh

```diff
@@ -46,6 +52,15 @@ trap cleanup EXIT
 
 prepare () {
     pip3 install -r $ACTION_PATH/requirements.txt
+
+    # Overwrite roles file if it exists already
+    echo "#!/bin/bash" > $ROLES_FILE_PATH
+    echo "set -e" >> $ROLES_FILE_PATH
+
+    for ROLE in ${REQUIRED_ROLES}; do
```

Comment:
Could you add this for-loop to [initdb.sh](https://github.com/mx51/data-dictionary-action/blob/master/containers/postgres/initdb.sh) and pass it the optional roles to action?

Reply:
Sure, happy to combine them.